js/main.js

```diff
@@ -0,0 +1,53 @@
+const NAMES_ARRAY = [
+  'Андрей','Мария',
+  'Станислав',
+  'Олег'
+];
+const COMMENTS_ARRAY = [
+  'Всё отлично!',
+  'В целом всё неплохо. Но не всё. Когда вы делаете фотографию, хорошо бы убирать палец из кадра.',
+  'Моя бабушка случайно чихнула с фотоаппаратом в руках и у неё получилась фотография лучше.'
+];
+const DESCRIPTION = [
+  'Это мяч',
+  'Коты наше все',
+  'Анапа 2008',
+  '#Я в коридоре на Балли'
+];
+const OBJ = [];
+const getRandomPositiveInteger = (a, b) =>{
+  const lower = Math.ceil(Math.min(a, b));
+  const upper = Math.floor(Math.max(a, b));
+  const result = Math.random() * (upper - lower + 1) + lower;
+  return Math.floor(result);
+};
+const createIdGenerator = () => {
+  let lastGeneratedId = 0;
+  return function (){
+    lastGeneratedId += 1;
+    return lastGeneratedId;
+  };
+};
+const randomIdNumber = createIdGenerator();
+const randomUrlNumber = createIdGenerator();
+const createObj = () => {
+  const randomDescriptionIndex = getRandomPositiveInteger(0, DESCRIPTION.length - 1);
+  const randomCounterLikes = getRandomPositiveInteger(15,200);
+  const randomCommentsIndex = getRandomPositiveInteger(0,COMMENTS_ARRAY.length - 1);
+  const randomNameIndex = getRandomPositiveInteger(0,NAMES_ARRAY.length - 1);
+  return {
+    description: DESCRIPTION[randomDescriptionIndex],
+    likes: randomCounterLikes,
+    comments: COMMENTS_ARRAY[randomCommentsIndex],
+    name: NAMES_ARRAY[randomNameIndex],
+  };
+};
+
+const saveValueIndexObj = () =>{
```

Comment:
Лучше переименовать в `createPhotoDescriptions` по аналогии с предыдущей, только предыдущая функция возвращала 1 объект описания фотографии, а эта будет возвращать массив таких описаний.

Да, понятно, в чем была идея изначально определить OBJ  и с помощью вызова этой функции наполнить его. Но  лучше отойти от этого и сделать функцию, внутри которой и создается искомый массив.
```suggestion
const createPhotoDescriptions= () =>{
  const photos = [];
```